src/app/Providers/components/ProvidersTable/VMware/InventoryDownload.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { LevelItem, Text } from '@patternfly/react-core';
+import { useQuery } from 'react-query';
+import { useAuthorizedFetch } from '@app/queries/fetchHelpers';
+import { VIRT_META } from '@app/common/constants';
+import { IVMwareProvider } from '@app/queries/types';
+
+interface IInventoryDowloadProps {
+  providers: IVMwareProvider[];
+}
+
+const InventoryDownload: React.FunctionComponent<IInventoryDowloadProps> = ({ providers }) => {
+  const useDownload = () => {
+    return useQuery({
```

Comment:
As you mentioned on Slack, we probably don't need react-query at all here, right? Since we're not catching it, all we need to do is have an async download function that will `const payloadBlob = await authorizedFetch(...);` and then write that to the file I think

Reply:
Right and that's why https://github.com/konveyor/virt-ui/pull/197 seems more straightforward.
Meanwhile to make is work with this approach that seems to be the way.

Reply:
@gildub hmm, now that I think of it we would be able to get error handling out of react-query.